tests/v2/08_full_workflow/test_full_workflow_local_v2.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from common.common import failing_workflow_UnknownError
+from common.common import full_workflow
+from common.common import full_workflow_TaskExecutionError
+from common.common import non_executable_task_command
+
+
+async def test_full_workflow_local(
+    client,
+    MockCurrentUser,
+    tmp777_path,
+    project_factory_v2,
+    dataset_factory_v2,
+    workflow_factory_v2,
+    override_settings_factory,
+    tmp_path_factory,
+    fractal_tasks_mock,  # needed
+):
+    # Use a session-scoped FRACTAL_TASKS_DIR folder
+    override_settings_factory(
+        FRACTAL_RUNNER_BACKEND="local",
+        FRACTAL_RUNNER_WORKING_BASE_DIR=tmp777_path / "artifacts-local",
```

Comment:
No need for `artifacts-local` any more, since this test is not parametrized over backend. We can keep it cleaner, with just `artifacts`.
(same applies to most other tests)